### ISSUE TITLE
Crash when reading classifier model `FluidMLPClassifier` from JSON file 

### ISSUE THREAD

**Issue by axambo** [flucoma/flucoma-sc#159]
Hello!

There's a crash when reading the classifier model `FluidMLPClassifier` from a JSON file. The SC post window returns:
```
Server 'localhost' exited with exit code 0.
server 'localhost' disconnected shared memory interface
```

To replicate the error, you can follow the `FluidMLPClassifier` Help file and add this code (you will need to change the paths):

```
~mlp.write("/Users/anna/Desktop/test/testMLP.json")

~mlp2 = FluidMLPClassifier(s,[3],activation:FluidMLPClassifier.sigmoid,maxIter:20,learnRate:0.01,batchSize:1,validation:0.1);

~mlp2.read("/Users/anna/Desktop/test/testMLP.json")

```

**Comment by tremblap** [comment]
hmmm here it doesn't crash. Which version have you got?  Can you try the 1.0.6 release candidate? I am testing on :

Fluid Corpus Manipulation Toolkit: version 1.0.6+sha.8403b25.core.sha.4a7d9daa

I have the previous public version (1.0.5) which I will try right away and hope to reproduce.

**Comment by tremblap** [comment]
ok I was able to reproduce in the previous public version (1.0.5+sha.5bb4245.core.sha.001df55a)
 - luckily, we fixed it in 1.0.6 it seems. Can you confirm?

**Comment by axambo** [comment]
I tried 1.0.6 (pre-release) and it works, thank you very much!

**Comment by tremblap** [comment]
fixed in forthcoming 1.0.6

**Comment by axambo** [comment]
Super, thanks! already using it :)
